src/main/kotlin/no/nav/familie/ef/iverksett/brev/BrevdistribusjonConflictExceptionResponseBody.kt

```diff
@@ -0,0 +1,3 @@
+package no.nav.familie.ef.iverksett.brev
+
+data class BrevdistribusjonConflictExceptionResponseBody(val bestillingsId: String)
```

Comment:
Kanskje er tilstrekkelig med `Brevdistribusjonskonflikt` ? 
```suggestion
data class Brevdistribusjonskonflikt(val bestillingsId: String)
```